.github/workflows/test.yml

```diff
@@ -0,0 +1,32 @@
+name: Test
```

Comment:
(commenting on the pull request as a way to explain what the code does)
This github action is a pretty vanilla approach that could be re-used in many node projects to automatically run their defined test suite on push and on pull requests.